examples/siro_sandbox/controllers/gui_controller.py

```diff
@@ -245,8 +247,16 @@ def get_random_joint_action(self):
         )
         return humanoidjoint_action
 
-    def set_act_hints(self, walk_dir, grasp_obj_idx, do_drop, cam_yaw=None):
+    def set_act_hints(
+        self,
+        walk_dir,
+        distance_multiplier,
+        grasp_obj_idx,
+        do_drop,
+        cam_yaw=None,
+    ):
         self._hint_walk_dir = walk_dir
+        self._hint_distance_multiplier = distance_multiplier
```

Comment:
Where is the logic that uses distance_multiplier?

Reply:
Oh, yes. I was experimenting with random quaternions and unintentionally rolled back the code with distance_multiplier.